androidApp/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="access_issue">Access Issue</string>
-    <string name="access_issue_sentence_case">Access issue</string>
+    <string name="access_issue" tools:ignore="MissingTranslation">Access Issue</string>
+    <string name="access_issue_sentence_case" tools:ignore="MissingTranslation">Access issue</string>
```

Comment:
@EmmaSimon I see there are placeholder translations for these ones missing translations on main. If the import removed them, is it possible to add them back with manual editing of the iOS strings file?

Reply:
Ah, got it! I was worried there was an issue with importing overwriting things, glad that isn't the case.